package.json

```diff
@@ -80,7 +80,7 @@
     "@types/uuid": "^3.4.4",
     "commitizen": "2.10.1",
     "cross-env": "^5.2.0",
-    "cypress": "^3.0.3",
+    "cypress": "^3.1.5",
```

Comment:
Oh yes, this requires an upgrade to the latest version of Cypress. Necessary to do a cy.writeFile with the {flag: 'a'} option in it. (appending to the file)